static/app/utils/replays/hooks/useIsLive.tsx

```diff
@@ -0,0 +1,24 @@
+import {useState} from 'react';
+
+import type ReplayReader from 'sentry/utils/replays/replayReader';
+
+type Props = {
+  replayReader: ReplayReader | null;
+};
+
+function useIsLive({replayReader}: Props) {
```

Comment:
nit: you can put the export on the same line up here. Makes it easy to know that this is the main thing in the file right away.

```suggestion
export default function useIsLive({replayReader}: Props) {
```